sap/src/main/java/org/frankframework/extensions/sap/jco3/SapListenerImpl.java

```diff
@@ -347,4 +347,10 @@ public void rollback(JCoServerContext serverCtx, String tid) {
 	protected String getFunctionName() {
 		return null;
 	}
+
+	// So the IDocXMLProcessor can be mocked during testing.
+	public IDocXMLProcessor getIDocXMLProcessor() {
```

Comment:
Als dit puur gebeurd in de sap package kan deze beter protected or package-private zijn.

Reply:
Hij moet ook vanaf de test module benaderbaar zijn en dat kan jammer genoeg niet als hij protected is.